drv/sprot-api/src/lib.rs

```diff
@@ -499,6 +510,9 @@ pub enum RspBody {
     Attest(Result<AttestRsp, AttestError>),
 
     Page(Result<RotPageRsp, UpdateError>),
+
+    // Added in sprot protocol version 6
+    State(Result<StateRsp, StateError>),
```

Comment:
We call these states `lifecycle states` in rfd 286. If it's not too many characters that help differentiate it from the `RotState` variant.